azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/CallCompositeBuilder.java

```diff
@@ -16,6 +17,10 @@ public final class CallCompositeBuilder {
 
     private Integer themeConfig = null;
     private CallCompositeLocalizationOptions localizationConfig = null;
+    private CallCompositeSupportedScreenOrientation callScreenOrientation =
```

Comment:
Please do not set default value here. Contoso may not pass anything, then CallCompositeConfiguration should have null for these values.

Reply:
I am setting values here incase Contoso does not pass anything. The values here represent our current default behaviour orientation for non-tv users.